primefaces/pom.xml

```diff
@@ -1226,7 +1226,7 @@
             <properties>
                 <maven.test.skip>true</maven.test.skip>
                 <maven.javadoc.skip>true</maven.javadoc.skip>
-                <maven.source.skip>true</maven.source.skip>
+                <maven.source.skip>false</maven.source.skip>
```

Comment:
Was this an accident flipping this flag?

Reply:
yes xD